src/Composer/Installers/BitrixInstaller.php

```diff
@@ -26,9 +26,12 @@
 class BitrixInstaller extends BaseInstaller
 {
     protected $locations = array(
-        'module'    => '{$bitrix_dir}/modules/{$name}/',
-        'component' => '{$bitrix_dir}/components/{$name}/',
-        'theme'     => '{$bitrix_dir}/templates/{$name}/',
+        'module'    => '{$bitrix_dir}/modules/{$name}/',    //deprecated, remove if the year >= 2018 on the major release (Backward compatibility will be broken)
```

Comment:
> deprecated, remove if the year >= 2018 on the major release (Backward compatibility will be broken)

It can deletes only if released major version :)